canary_tests.py

```diff
@@ -98,7 +98,11 @@ async def test_interrupts_and_pwm(self, pwm):
         should_stop = asyncio.Event()
         ticks = []
         tick_stream = await self.board.stream_ticks([interrupt])
-        counter_task = asyncio.create_task(self.record_tick_data(tick_stream, ticks, should_stop))
+
+        # If we are not getting any ticks in tick_stream the task will wait 5 secs
+        # for record_tick_data to return, then timeout.
+        counter_task = asyncio.create_task(asyncio.wait_for(\
```

Comment:
nit: no need for the backslash. Those are discouraged, and only needed if you aren't inside a parenthesis.

Reply:
oooo, that's good to know.